### ISSUE TITLE
Бартерон, встроенный внутри приложения Бастион, работает нестабильно

### ISSUE THREAD

**Issue by Voyager-3rd** [pocketnetteam/barteron.gui#133]
Бартерон, встроенный внутри приложения Бастион, работает нестабильно. При переходе на страницу "Домашняя" не всегда показываются новинки. Если сделать переход к категории, то не всегда подгружается список, хотя точно известно, что категория непустая, например, "Продукты и напитки". См. первый скрин.

Получить точный запрос, который выполняется из приложения на бекенд оказалось проблематично. Дело в том, что если внутри приложения открыть Developer Tools (см. второй скрин) и список Network, то ключевой запрос (из списка запросов RPC) не попадает в данный список. Приложение Charles (https://www.charlesproxy.com) для и проксирования и отладки запросов тоже не ловит данный запрос. Исключением является случай использования бартерона через сайт, но это не та ситуация, которая требуется и проблем в этом случае замечено не было.

_Скрин 1_
<img width="1050" alt="Снимок экрана 2024-10-28 в 15 29 24" src="https://github.com/user-attachments/assets/4ac7befc-5540-43ca-b298-d032fb231492">

_Скрин 2_
<img width="884" alt="Снимок экрана 2024-10-28 в 15 24 13" src="https://github.com/user-attachments/assets/ff0ee6ae-3be7-494b-a51a-79153233ca4c">

**Comment by Voyager-3rd** [comment]
@andyoknen, подскажите, какие еще есть способы получить параметры запроса к бекенду с учетом деталей, описанных в задаче?

**Comment by andyoknen** [comment]
@Voyager-3rd т.е. проблема только в приложении? В браузере не замечено? В приложении запросы к нодам происходят во внутреннем процессе nodejs, к которому нет доступа из DevTools. Очевидно проблема в запросах к версиям нод 0.22.6

**Comment by Voyager-3rd** [comment]
@andyoknen, да, проблема наблюдается только в приложении. В браузере ни разу данного сбоя не наблюдалось. В коде явно указано обращение к https://2.pocketnet.app:8899/ и по идее, вне зависимости от режима работы (браузер или приложение), запросы должны идти на этот адрес. Затем, как я понимаю, выбирается конкретная нода. Почему получается разный результат браузера и приложения пока непонятно. Но проблема представляется критической, т.к. сильно затрудняет использование бартерона через приложение.
